k8s/migration/pkg/utils/credutils.go

```diff
@@ -665,7 +713,21 @@ func GetAllVMs(ctx context.Context, k3sclient client.Client, vmwcreds *vjailbrea
 		} else {
 			clusterName = ""
 		}
-
+		if skipVm {
+			continue
+		}
+		if len(rdmDiskInfos) >= 1 && len(disks) == 0 {
+			ctxlog.Info("VM has multiple RDM disks but no regular bootable disks found", "vm", vm.Name(), "hence VM cannot be migrated")
+			continue
+		}
+		var rdmDisks []vjailbreakv1alpha1.RDMDiskInfo
+		if len(rdmDiskInfos) > 0 {
+			rdmDisks, err = PopulateRDMDiskInfoFromAttributes(ctx, rdmDiskInfos, attributes)
```

Comment:
```suggestion
			rdmDiskInfos, err = PopulateRDMDiskInfoFromAttributes(ctx, rdmDiskInfos, attributes)
```
Similar to append operation, I think we can rather rely on this and get rid of extra `rdmDisks` variable?

You need to remove `rdmDisks` variable, I'm suggesting single change.

Reply:
we have len check on rdmDiskInfo , hence we cannot reuse same variable , lint throws below error
![Screenshot from 2025-06-11 17-44-43](https://github.com/user-attachments/assets/291d9516-c4ad-49b8-9f3d-2abb8907558e) assignment to rdmDiskInfos (ineffassign)`

Reply:
This issue was happening I guess because we were not using this variable. In previous changes only this line was changed and rdmDisks was getting used making this rdmDiskInfos unused. 
Can you do the complete change and see what happens?